src/mcts/search.cc

```diff
@@ -806,6 +812,19 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
 
   // Fetch the current best root node visits for possible smart pruning.
   const int64_t best_node_n = search_->current_best_edge_.GetN();
+  // and Q for moves_left logic.
+  const float best_node_q = search_->current_best_edge_.GetQ(0);
+
+  bool use_moves_left = moves_left_support_ && params_.GetUseMovesLeft();
+  int moves_left = 0;
+  float moves_threshold = params_.GetMovesLeftThreshold();
+  float best_node_m = search_->current_best_edge_.GetM(0.0f);
```

Comment:
Wouldn't it be better to use the parent's `GetM()` estimation here? It will still be dominated by the best child, but without jumps when the best child changes, and with the benefit of rewarding shorter wins/longer losses here as well.

Reply:
Makes sense. It could be tested.

Reply:
Parent's M is used in the newest version.